<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="text_output">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ano">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Año inicial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sigla">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigla del curso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graduados">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Graduados/región</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_notas">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Estudiantes/nota</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nota_min">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota mínima</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_sigla">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_canton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_distrito">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_distrito">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distrito</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_canton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantón</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_provincia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Provincia</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_provincia">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_nota_min">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_nota_max">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_nota_max">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota máxima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ano_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Año final</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_ano_final">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_ano_inicial">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>204</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_username"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button_connect">
       <property name="text">
        <string>Log in</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota/curso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_carne">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_carne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Carné</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>button_connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_graduados</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>graduados_region()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_notas</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nota_curso_limitada()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dropdown_provincia</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>fill_list_canton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dropdown_provincia</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>fill_list_canton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dropdown_canton</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>fill_list_distrito()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dropdown_canton</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>fill_list_distrito()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>promedio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>promedio_estudiante()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connect()</slot>
  <slot>create_tables()</slot>
  <slot>graduados_region()</slot>
  <slot>nota_curso_limitada()</slot>
  <slot>fill_list_canton()</slot>
  <slot>fill_list_distrito()</slot>
  <slot>promedio()</slot>
  <slot>promedio_estudiante()</slot>
 </slots>
</ui>
